### ISSUE TITLE
Apple Card card design is not readable

### ISSUE THREAD

**Issue by henrysha** [henrysha/credit-card-rewards-pwa#43]
<img width="1290" height="659" alt="Image" src="https://github.com/user-attachments/assets/e34cbf5a-76af-4bba-aac1-8b0216718781" />

The contrast between background and text is too low, and it makes it very hard to read. 

Add to UI constraints rule to comply to WCAG 2.0 standards as well.
